spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/CAddTable.scala

```diff
@@ -38,6 +41,31 @@ class CAddTable[T: ClassTag, D: ClassTag](val inplace: Boolean = false)(
 
   output = Tensor[D]()
 
+  private def canFastBroadcast(tensor: Tensor[D], other: Tensor[D]): Boolean = {
```

Comment:
what does `canFastBroadcast` mean ?

Reply:
We need to handle two tensors with different size. If they have same dims, and size(dim) is same or one of them is 1, that means tensors can be added together.

Reply:
ok, but why we can `canFastBroadcast` ? the naming is confusing

Reply:
This name is refered to same operation in tensor add. 
How about `canBroadcast` or `canExpand` ?

Reply:
The method here is we want to add along all dims with 1 and then to see it can be the same size as the reference one right ? need to think how to name it...